doc/dev_manual/code/io_hdf5.py

```diff
@@ -75,11 +79,11 @@ def load_idx(self, idx, **kwargs):
             from ginga import AstroImage, RGBImage
 
             header = self.get_header(idx)
-            data_np = np.copy(self._f[idx].value)
+            data_np = np.copy(self._f[idx][()])
```

Comment:
I don't understand passing in empty parenthesis as a subscript. Is this HDF5 specific usage?

Reply:
Yes, I got a deprecation warning with the earlier syntax.  This was suggested on the h5py site.  I tested it and it works fine.